hack/make.sh

```diff
@@ -145,7 +145,7 @@ ORIG_BUILDFLAGS=( -a -tags "autogen netgo static_build sqlite_omit_load_extensio
 # see https://github.com/golang/go/issues/9369#issuecomment-69864440 for why -installsuffix is necessary here
 BUILDFLAGS=( $BUILDFLAGS "${ORIG_BUILDFLAGS[@]}" )
 # Test timeout.
-: ${TIMEOUT:=120m}
+: ${TIMEOUT:=180m}
```

Comment:
Sure, perhaps we can tweak that a little bit. But my last Jenkins tests start failing after this timeout. Even 150m made some trouble.
Perhaps we should set the TIMEOUT explicit in the arm build steps, so the default here could be lowered again.